### DISCUSSION TITLE
pivpn -c client "last seen" list expires

### DISCUSSION THREAD

**Discussion by williamthrilliam** [pivpn/pivpn#1355] [Feature requests]
It would be nice if the connected client list showed the last connection time and it never expired. I haven't monitored the expiration time, but after a week or so the results for pivpn -c show "last seen not yet" for clients that were previously connected.

**Comment by orazioedoardo** [comment]
Sorry for the huge delay. If you are still interested in this: `pivpn -c` extracts the output from `wg show`, and that's in RAM. If you restart wireguard or the Raspberry, it's reset.
